auth/src/test/java/com/firebase/ui/auth/test_helpers/FacebookProviderShadow.java

```diff
@@ -44,8 +41,6 @@ public FacebookProviderShadow() {
         }
     }
 
-    public void __constructor__(Activity activity, List<String> scopes) {}
```

Comment:
This is actually used to prevent the real constructor from being called.  Should have left a comment, sorry.